mlflow/pytorch/__init__.py

```diff
@@ -343,24 +388,55 @@ def save_model(
     .. code-block:: python
         :caption: Example
 
+        import os
+
+        import numpy as np
         import torch
-        import mlflow
         import mlflow.pytorch
-        # Create model and set values
-        pytorch_model = Model()
-        pytorch_model_path = ...
-        # train our model
-        for epoch in range(500):
-            y_pred = pytorch_model(x_data)
+
+        # Class defined here
+        class LinearNNModel(torch.nn.Module):
             ...
-        # Save the model
+
+        # Initialize our model, criterion and optimizer
+        ...
+
+        # Training loop
+        ...
+
+        # Save PyTorch models to current working directory
         with mlflow.start_run() as run:
-            mlflow.log_param("epochs", 500)
-            mlflow.pytorch.save_model(pytorch_model, pytorch_model_path)
+            mlflow.pytorch.save_model(model, "models_pth")
 
-            # Saving scripted model
+            # Convert to a scripted model and save it
             scripted_pytorch_model = torch.jit.script(model)
-            mlflow.pytorch.save_model(scripted_pytorch_model, pytorch_model_path)
+            mlflow.pytorch.save_model(scripted_pytorch_model, "scripted_models_pth")
+
+        # Load each saved model for inference
+        for model_path in ["models_pth", "scripted_models_pth"]:
```

Comment:
Why do we suffix directorie with `_pth`?

Reply:
Good observation and point! No need to adopt the PyTorch suffix for model directories. We use generic "models," so better to stick with "models" and scripted_models" respectively.